lib/sqitch-passwords.pod

```diff
@@ -75,6 +75,10 @@ Firebird supports passwordless authentication only via
 L<trusted authentication|http://www.firebirdsql.org/manual/qsg2-config.html>
 for local connections.
 
+=item Exasol
+
+Exasol doesn't seem to support password-less authentication at this time.
```

Comment:
Should link to documentation for configuring Exasol authentication. Does not have to support passwordless.